### PR TITLE
Fixed amazon-cloud-drive cask.

### PR DESCRIPTION
**Pull request by egyptianbman**

### CODE REVIEW

File: Casks/amazon-cloud-drive.rb [hunk 1]
```diff
@@ -4,9 +4,9 @@
 
   # cloudfront.net is the official download host per the vendor homepage
   url 'https://d29x207vrinatv.cloudfront.net/mac/AmazonCloudDriveInstaller.dmg'
-  name 'Amazon Cloud Drive App'
+  name 'Amazon Cloud Drive Installer'
   homepage 'https://www.amazon.com/clouddrive/home/'
   license :gratis
 
-  app 'Amazon Cloud Drive App.app'
+  app 'Amazon Cloud Drive Installer.app'
```
Comment (vitorgalvao) [inline Casks/amazon-cloud-drive.rb]: Is this an installer you need to run, now? Then it needs to use [installer :manual](https://github.com/caskroom/homebrew-cask/blob/master/doc/CASK_LANGUAGE_REFERENCE.md#installer-manual) and requires It’s missing [`uninstall`](https://github.com/caskroom/homebrew-cask/blob/master/doc/CASK_LANGUAGE_REFERENCE.md#uninstall-stanza-details).

Comment (Ngrd) [inline Casks/amazon-cloud-drive.rb]: Can confirm that the installer is used now. You can use this to automate the installation and uninstallation processes for this cask:
```ruby
cask :v1 => 'amazon-cloud-drive' do
  version :latest
  sha256 :no_check

  # cloudfront.net is the official download host per the vendor homepage
  url 'https://d29x207vrinatv.cloudfront.net/mac/AmazonCloudDriveInstaller.dmg'
  name 'Amazon Cloud Drive App'
  homepage 'https://www.amazon.com/clouddrive/home/'
  license :gratis

  installer :script => 'Amazon Cloud Drive Installer.app/Contents/MacOS/Amazon Cloud Drive Installer'

  uninstall :quit => 'com.amazon.clouddrive.mac',
            :delete => '/Applications/Amazon Cloud Drive App.app'
end
```
However, I didn't find any arguments that can be passed to the installation script to run it quietly.

Comment (egyptianbman) [inline Casks/amazon-cloud-drive.rb]: Thank you for providing that code.  I can confirm that it does install automatically now.  I couldn't get any information out of the executable.  Would you like me to update the PR or continue to look for a way to make it quiet?

Comment (Ngrd) [inline Casks/amazon-cloud-drive.rb]: I guess it's ok that this script doesn't run quietly. You can simply update your PR to include all the necessary changes. You can always open a new PR if you find a way to improve the current solution.

Comment (vitorgalvao) [inline Casks/amazon-cloud-drive.rb]: @egyptianbman Please @mention any of us after you update, so we get an email to check back.

**Comment by egyptianbman** [comment]
@vitorgalvao @Ngrd I've updated the PR; thanks.

**Comment by vitorgalvao** [comment]
Thank you. Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by egyptianbman** [comment]
Squashed, thanks!

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It was merged directly as d41964e1198531e6ca931690f28e66da061736f7 to keep commit history cleaner. Your contribution is still credited to you.

**Pull request closed without merging**